Project doesn't build due to missing Jitpack Repositories

Stacktrace:
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':myairplanefork-server:shadowJar'.
> Could not resolve all dependencies for configuration ':myairplanefork-server:runtimeClasspath'.
   > Could not find com.github.technove:AIR:fe3dbb4420.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/github/technove/AIR/fe3dbb4420/AIR-fe3dbb4420.pom
       - https://repo.aikar.co/content/groups/aikar/com/github/technove/AIR/fe3dbb4420/AIR-fe3dbb4420.pom
       - https://nexus.velocitypowered.com/repository/velocity-artifacts-snapshots/com/github/technove/AIR/fe3dbb4420/AIR-fe3dbb4420.pom
       - https://libraries.minecraft.net/com/github/technove/AIR/fe3dbb4420/AIR-fe3dbb4420.pom
       - file:/home/phil/.m2/repository/com/github/technove/AIR/fe3dbb4420/AIR-fe3dbb4420.pom
     Required by:
         project :myairplanefork-server
   > Could not find com.github.technove:Flare:master-SNAPSHOT.
     Searched in the following locations:
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -39,6 +39,7 @@ subprojects {
         maven("https://repo.aikar.co/content/groups/aikar/")
         maven("https://nexus.velocitypowered.com/repository/velocity-artifacts-snapshots/")
         maven("https://libraries.minecraft.net")
+        maven("https://jitpack.io")
         mavenLocal()
     }
 

